pkg/controller.v1/xgboost/status.go

```diff
@@ -1,32 +0,0 @@
-package xgboost
-
-import (
-	"fmt"
-
-	"github.com/sirupsen/logrus"
-	corev1 "k8s.io/api/core/v1"
-
-	kubeflowv1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
-	commonutil "github.com/kubeflow/training-operator/pkg/util"
-)
-
-func setRunningCondition(logger *logrus.Entry, jobName string, jobStatus *kubeflowv1.JobStatus) error {
```

Comment:
This function doesn't need since we can do the same process using the `commonutil.UpdateJobConditions`.